CharacterCreation/07_Items.txt

```diff
@@ -2,164 +2,178 @@
-Restores 2d10 + 20 nanites.
+    If you have Skill in Repair, see the skill sheet for separate instructions.
+
+Nanite Pack ($100):
+    Restores 2d10 + 20 nanites.
 
-Security Spike: $100
-+20 to a hack check on a door with a physical access panel. DC 30 to place. 
+Security Spike ($100):
+    +20 to a hack check on a door with a physical access panel. DC 30 to place. 
 Once hacked the door can be controlled remotely.
 
-Medic Needle Pack: $60
-Contains 10 needles suitable for use with several malpractice proceedures.
+Medic Needle Pack ($60):
+    Contains 10 needles suitable for use with several malpractice proceedures.
 Not recomended for medicianal use. 
 
-Single Use door brute-forcer $120:
-Hacks through the encryption on a door (up to 75 DC) and opens it.
+Door Brute-Forcer ($120):
+    Hacks through the encryption on a door (up to 75 DC) and opens it. Single 
+use device.
 
 Zip-Cuffs ($25):
-pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
+    Pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
 to break.
 
 HD Zip-Cuffs ($300):
-Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
+    Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
 
 ==============================
 Explosives
 ==============================
-Breaching charges ($500):
-Will remove a door from your path. Make sure no one is within 2m of the door
+Breaching charges ($160):
+    Will remove a door from your path. Make sure no one is within 2m of the door
 in any direction or they take damage as if hit by an ECSC fragmentation 
 grenade.
 
-Wall Charge ($840):
-Reduces a 1.5x1.5m section of wall to rubble, allowing entry. Clear areas 2m 
-surrounding the charge or take 70 explosive damage.
-Wall thicknesses:
-Wood - Up to 12in.
-concrete - up to 10in
-Reinforced Concrete - up to 8 in
-Steel plate - up to 2 in
+Wall Charge ($120):
+    Reduces a 1.5x1.5m section of wall to rubble, allowing entry. Clear areas 2m 
+surrounding the charge or take 70 explosive damage. Thicknesses of several 
+materials one wall charge can breach through are listed below.
 
-Det Cord ($230 per 2m of cord):
-Used to remove doors from hinges, act as ignition sources, enhance another 
-explosive, etc. Use 2m per door to be removed. .5m to wrap around a door 
-handle. 1m to wrap around a grenade. 4m to create a wall breaching pad (
-Explosives 50 DC)
+Wood:       Up to 12in.
+Concrete:   up to 10in
+Reinforced Concrete: up to 8 in
+Steel plate: up to 2 in
 
+Det Cord ($230 per 2m of cord):
+    Used to remove doors from hinges, act as ignition sources, enhance another 
+explosive, etc. Use 2 meters per door to be removed. One half meter to wrap 
+around a door handle. 1 meter to wrap around a grenade. 4 meters to create a 
+wall breaching pad (Explosives 50 DC).
 
+==============================
+Miscellaneous Items
+==============================
 
 Misc. Items and RP trinkets:
```

Comment:
Blocker: redundant heading. Remove the topic one, keep the chapter one for consistency with the rest of the document.

Reply:
wilco